### ISSUE TITLE
jsx-wrap-multiline false positive with React.Fragment shorthand tag

### ISSUE THREAD

**Issue by Kovensky** [palantir/tslint-react#124]
```tsx
const jsx = (
  <>
    <div>
      Test
    </div>
  </>
)
```

The above snippet causes a false positive warning from `jsx-wrap-multiline` -- it wants the `<div>` child to be wrapped with parens, however this is not necessary as the parent is already jsx.

I don't know how the rule works, but probably `jsx-wrap-multiline` just needs to also accept the new AST node type as also being JSX.

`<>` / `</>` support was released on TS 2.6.2, and supported at runtime by React 16.2.0.
